src/utils/__tests__/array.test.js

```diff
@@ -18,7 +18,7 @@ describe('array utils', () => {
       const subject = null;
       expect(() => {
         subject.findLastIndex((el) => el === 'a');
-      }).toThrow('Cannot read property \'findLastIndex\' of null');
+      }).toThrow();
```

Comment:
This is the specific change I refer to in the PR description, the test above is just a test for when only *one* element satisfies the predicate, which was missing.